### PR TITLE
Fixes HE pipes not respecting layering

### LINKED ISSUE #38467: h/e junction ignores layer bounderies
Teset server compiled on 6/13/18

Any h/e junction will connect to a pipe parallel to it regardless of layer 

[Reproduction]: place a heat exchange junction on a separate layer inline with any atmospherics pipeline
image

### PR DESCRIPTION
**Pull request by silicons**
Forgot that they override this proc, woops!

---

Fixes #38467.

### CODE REVIEW

File: code/modules/atmospherics/machinery/pipes/heat_exchange/he_pipes.dm [hunk 1]
```diff
@@ -15,9 +15,12 @@
 
 /obj/machinery/atmospherics/pipe/heat_exchanging/can_be_node(obj/machinery/atmospherics/pipe/heat_exchanging/target)
 	if(!istype(target))
-		return 0
+		return FALSE
+	. = ..()
+	if(!.)
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/atmospherics/machinery/pipes/heat_exchange/he_pipes.dm]: Why are you not just `if(..())`?

**Comment by ExcessiveUseOfCobblestone** [comment]
Is this the issue with why my massive Space HE setups with layering were causing SM choking?

*angry moo*

**Comment by kevinz000** [comment]
gonna take a better looka t the code in like half an hour gonna just mark this wip as it is getting tested when i do that

**Comment by AnturK** [comment]
If this is in progress what's the progress.

**Pull request merged**